controllers/clientsController.js

```diff
@@ -36,24 +36,21 @@ router.put('/', function(req,res){
 	}
 })
 
-
-// Delete a Client, only signed user
-// router.delete('/id', function(req, res){
-// 	clients.findone({id: 2}, (err, result) => {
-// 		if(result) {
-// 			clients.deleteone({id: 2})
-// 			 .then((err) => {
-// 			 	res.status(200).json({ result: 'ok'})
-// 			 })
-// 			 .catch(err => {
-// 			 	res.status(500).json({ message: 'Something went wrong', error: err})
-// 			 })
-// 		}
-// 		else {
-// 			res.json({ error: 'id not found'})
-// 		}
-// 	})
-// })
-
+// Delete a Client by ID
+router.delete('/', function(req,res) {
+	let idFound = ClientModel.findById(req.client.id)
```

Comment:
this validation to check if exists or not should be in the Model side, and return true/false if it found and was success

Reply:
My process was sending the client ID to the model side and if it finds the ID, it will return true (or anything). In the controller, it calls this method, if it returns something (which is idfound), it goes to the next step to delete. If it doesn't return anything, it gives the status 404 not found. that's why I have 3 stages in this delete request. This is similar to what you did in your program reference.

Reply:
yes yes, and it's ok but in the case of the delete the validation should be in the model side (tomo we can change together if you want) 

the controller should have only the responsibility to call model, and return the correct message

Reply:
sure! thanks! and btw, what do you mean by "Create an issue to view a single one" in the training documentation? I sent you an email but maybe you didn't see it.

Reply:
is to get by id only 1 instead of all.

example `url/clients/3` should get a single one with id 3

Reply:
got it! thank you!